### PR TITLE
Fix #1987: optimize code by ensuring all the TextViews in ongoing_story_card.xml use style from styles.xml"

### LINKED ISSUE #1987: Optimize code in ongoing_story_card.xml
This issue is tracking the optimization work of `ongoing_story_card.xml`

1. Make sure that the layout uses `ConstraintLayout` and there is minimal nesting of views. If this view already uses `ConstraintLayout` then it's fine.

2. Make sure that all TextViews use the style from `styles.xml` file like `Heading1`, `TextFieldLabel`, `Body`, etc. whichever suits best.

While making the above changes the UI should not get affected and the end result should still be the same as before (or an optimized version of UI).

If there are multiple files with the same name, update them all.

## References
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by Florence-Njeri**
- Fixes #1987: Optimize code in ongoing_story_card.xml by adding style in styles.xml to all the TextViews
  - Reviewer - @rt4914 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by aggarwalpulkit596 — APPROVED** [review]
LGTM

File: app/src/main/res/layout-land/ongoing_story_card.xml [hunk 1]
```diff
@@ -92,3 +87,4 @@
     </androidx.constraintlayout.widget.ConstraintLayout>
   </com.google.android.material.card.MaterialCardView>
 </layout>
+
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout-land/ongoing_story_card.xml]: Revert this.

File: app/src/main/res/layout-sw600dp-land/ongoing_story_card.xml [hunk 2]
```diff
@@ -93,3 +88,4 @@
     </androidx.constraintlayout.widget.ConstraintLayout>
   </com.google.android.material.card.MaterialCardView>
 </layout>
+
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout-sw600dp-land/ongoing_story_card.xml]: Revert this.

File: app/src/main/res/layout-sw600dp-port/ongoing_story_card.xml [hunk 3]
```diff
@@ -93,3 +88,4 @@
     </androidx.constraintlayout.widget.ConstraintLayout>
   </com.google.android.material.card.MaterialCardView>
 </layout>
+
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout-sw600dp-port/ongoing_story_card.xml]: Revert this

File: app/src/main/res/layout/ongoing_story_card.xml [hunk 4]
```diff
@@ -93,3 +88,4 @@
     </androidx.constraintlayout.widget.ConstraintLayout>
   </com.google.android.material.card.MaterialCardView>
 </layout>
+
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/ongoing_story_card.xml]: Revert this

File: app/src/main/res/layout-sw600dp-land/ongoing_story_card.xml (hunk 2 above)
Comment (Florence-Njeri) [reply app/src/main/res/layout-sw600dp-land/ongoing_story_card.xml]: @aggarwalpulkit596 I have reverted the file to only contain the one extra line at the end as it was in the last versioned commit by Ben Henning. I hope that is what you needed to be reverted.

File: app/src/main/res/layout-land/ongoing_story_card.xml (hunk 1 above)
Comment (Florence-Njeri) [reply app/src/main/res/layout-land/ongoing_story_card.xml]: @aggarwalpulkit596  Done. Please review the changes made

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

@aggarwalpulkit596 Please resolve your comments and then we can merge this PR. Thanks.

**Pull request merged**